WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -453,7 +452,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                             return;
                         }
 
-                        updateFeaturedImage(Integer.parseInt(ids.get(0)));
+                        updateFeaturedImage(Long.parseLong(ids.get(0)));
```

Comment:
Do you think it's worthwhile using `StringUtils.stringToLong()` here instead, since `Long.parseLong()` can throw an exception?